src/Backend/Core/Engine/Url.php

```diff
@@ -253,9 +253,9 @@ private function getInterfaceLanguage(): string
             return Authentication::getUser()->getSetting('interface_language', $default);
         }
 
-        if (CommonCookie::exists('interface_language')) {
+        if ($this->getContainer()->get('fork.cookie')->has('interface_language')) {
```

Comment:
Typehinting on the service name works, but if you first get the container, followed by the service you get typehinting on the methods on that service